cla_public/templates/contact.html

```diff
@@ -25,7 +31,7 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
       to question 'Are you at immediate risk of harm?'
     #}
     {% if 'n18' in session.checker.diagnosis_previous_choices %}
-      <div class="govuk-warning-text">
+      <div class="govuk-warning-text"> 
```

Comment:
Cheeky bit of whitespace
```suggestion
      <div class="govuk-warning-text">
```